Multiplatform/Collections/Playlists/PlaylistAddSheet.swift

```diff
@@ -23,6 +24,12 @@ struct PlaylistAddSheet: View {
     
     @State private var playlists: [Playlist]?
     
+    init(track: Track){
```

Comment:
You can just use
`@State private var isPublic = !Defaults[.newPlaylistDefaultPrivate]`

Reply:
Resolved in d77f481 (used ai to write the commit message because I was a bit lazy haha 😅)